aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -594,6 +605,28 @@ protected boolean isDBInstanceStabilizedAfterMutate(
                 isMasterUserSecretStabilized(dbInstance);
     }
 
+    protected boolean isInstanceStabilizedAfterReplicationStop(final ProxyClient<RdsClient> rdsProxyClient,
+                                                               final ResourceModel model) {
+        final DBInstance dbInstance = fetchDBInstance(rdsProxyClient, model);
+
+        assertNoTerminalStatus(dbInstance);
+        return isDBInstanceAvailable(dbInstance)
+                && !dbInstance.hasDbInstanceAutomatedBackupsReplications();
```

Comment:
Let's introduce isDBInstanceHasBackupReplications() so we can reuse it here and in the method below.